src/main/java/com/github/bartimaeusnek/cropspp/GTHandler/machines/CropGeneExtractor.java

```diff
@@ -42,7 +42,7 @@ public CropGeneExtractor(int aID, String aName, String aNameRegional, int aTier)
                         "1 for Specimen, 2 for Growth, 3 for Gain, 4 for Resistance", "Takes in 1A",
                         "Needs crop's (tier+2)/2 as Voltage level, round down (Tier 5 crop needs 7/2=~3=HV)",
                         "Can process crops up to tier " + getMaxCropTier(aTier) },
-                2,
+                1,
```

Comment:
Decreasing the number of slots results in voiding existing item

Reply:
I know, but I highly doubt someone is storing their most prized seed bag in the machine meant to extract them. So at most we would be voiding a circuit, which I deemed to be worth it over keeping a redundant slot.

Also I think I would have to restrict the second slot to only circuits anyways so people don't put 2 seed bags in there and accidentally void one of them. I can however look into that.

Reply:
I see